### PR TITLE
Review Asset Status Mapping

### PR DESCRIPTION
**Pull request by lasato-OSI**

### CODE REVIEW

File: Content_Portal/Documentation/Assets/Asset-Type.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Content-Type: application/json
 }
 
 ```
+<! -- Replace "attribute" with "metadata" in the Description? -->
```
Comment (atangOSIsoft) [inline Content_Portal/Documentation/Assets/Asset-Type.md:70]: Yup!

File: Content_Portal/Documentation/Assets/Asset-Type.md [hunk 2]
```diff
@@ -160,6 +161,8 @@ To create an asset type with a specific ID, use the API route with ID. If this i
 }
 ```
 
+<! -- Should "Attributes" be changed to "Metadata" above? And should we use "metadata" in the Description?-- as in "This metadata indicates the model number ...." -->
```
Comment (atangOSIsoft) [inline Content_Portal/Documentation/Assets/Asset-Type.md:164]: Yup, I might have this in my PR already.

File: Content_Portal/Documentation/Assets/asset-status-mapping.md [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+
+## Status mapping properties table
+
+The following table lists the most common fields in a status mapping.
+
+| Property            | Type                     | Required? | Searchable? | Description                                                  |
+| ------------------- | ------------------------ | --------- | ----------- | ------------------------------------------------------------ |
+| Name                | String                   | Required  | No          | Name and ID for this status mapping.                         |
+| Description         | String                   | Required  | No          | Description of the status mapping.                           |
+| StreamReferenceId   | String                   | Required  | No          | The SDS stream reference for this status mapping, the property of this SDS stream is which the status will be derived from. <! -- I'm not sure what this sentence means. --> The stream reference must exist before the status mapping can be created. |
```
Comment (atangOSIsoft) [inline Content_Portal/Documentation/Assets/asset-status-mapping.md:19]: I can see how the sentence is confusing. The StreamReferenceId field refers to the Id for the StreamReference property on the Asset.  Let me know if this flows better "Refers to the Id of a StreamReference property on the Asset."

File: Content_Portal/Documentation/Assets/asset-status-mapping.md [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+| StreamPropertyId    | String                   | Required  | No          | The SDS stream property reference contained in the stream reference ID for this status mapping, the data of this SDS stream property is which the status will be derived from. <! -- I'm not sure what this sentence means. --> The SDS stream property (?) must exist before the status mapping can be created. |
```
Comment (atangOSIsoft) [inline Content_Portal/Documentation/Assets/asset-status-mapping.md:20]: Does this make more sense? 

"The Sds stream property that status will use for calculations.  This property must be present on the "StreamId" property on the Asset StreamReference."

The example below demonstrates this, so, maybe it's worth it to move the example up?

File: Content_Portal/Documentation/Assets/asset-status-mapping.md [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+| ValueStatusMappings | List<ValueStatusMapping> | Required  | No          | The value status mapping maps values to a given status. See [Value status mapping properties table](xref:AssetStatusMapping#value-status-mapping-properties-table) |
+
+## Value status mapping properties table
+
+The following table lists the most common fields in a value status mapping. A single value status mapping corresponds to a single status. If you want additional statuses in your asset status mapping ....<! -- Sentence not completed. -->
```
Comment (atangOSIsoft) [inline Content_Portal/Documentation/Assets/asset-status-mapping.md:25]: Complete sentence with "add additional elements in the ValueStatusMappings list."

File: Content_Portal/Documentation/Assets/asset-status-mapping.md [hunk 6]
```diff
@@ -0,0 +1,341 @@
...
+
+The asset type identifier
+
+### Response
+
+The response includes a status code and a response body.
+
+| Status Code     | Body Type                                                    | Description                                                  |
+| --------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| 200 OK          | `MeasurementMapping`<! -- This is MeasurementMapping, not MeasurementsMapping, right? --> | The requested status mapping of the specified asset or asset type. |
```
Comment (atangOSIsoft) [inline Content_Portal/Documentation/Assets/asset-status-mapping.md:122]: Correct, sorry, copy and paste error

File: Content_Portal/Documentation/Assets/asset-status-mapping.md [hunk 7]
```diff
@@ -0,0 +1,341 @@
...
+```
+
+### Response
+
+The response includes a status code and a response body.
+
+| Status Code      | Body Type         | Description                                                  |
+| ---------------- | ----------------- | ------------------------------------------------------------ |
+| 200 OK           | Asset status list | On successful POST, returns the status of multiple assets.   |
+| 207 Multi Status | Multi Status      | On POST, returns the status of multiple assets. The asset status in error, look at multi-status response. <! -- I don't understand this sentence. --> |
```
Comment (atangOSIsoft) [inline Content_Portal/Documentation/Assets/asset-status-mapping.md:309]: Multi status responses will response with an array of responses. Successful responses will just have what the user requested, for error responses, it'll show the error and reason. Perhaps you can help revise it if it makes sense?

File: Content_Portal/Documentation/Assets/toc.yml [hunk 8]
```diff
@@ -1,12 +1,16 @@
-- name: AssetType API
-  href: Asset-Type.md
 - name: Assets API
   href: Assets-API.md
-- name: Asset and AssetType Access Control API
-  href: Access-Control-API.md
-- name: Asset Measurements and MeasurementsMapping
+- name: AssetTypes API
+  href: Asset-Type.md
```
Comment (atangOSIsoft) [inline Content_Portal/Documentation/Assets/toc.yml:4]: maybe Asset-Types-API.md to make consistent with Assets.md?

File: Content_Portal/Documentation/Assets/asset-status-mapping.md (hunk 3 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-status-mapping.md:19]: How about: ID for the asset's StreamReference property.

File: Content_Portal/Documentation/Assets/Asset-Type.md (hunk 2 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/Asset-Type.md]: done

File: Content_Portal/Documentation/Assets/Asset-Type.md (hunk 1 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/Asset-Type.md]: done

File: Content_Portal/Documentation/Assets/asset-status-mapping.md (hunk 4 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-status-mapping.md:20]: done.

File: Content_Portal/Documentation/Assets/asset-status-mapping.md (hunk 5 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-status-mapping.md:25]: done

File: Content_Portal/Documentation/Assets/asset-status-mapping.md (hunk 6 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-status-mapping.md:122]: done

File: Content_Portal/Documentation/Assets/asset-status-mapping.md (hunk 7 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-status-mapping.md:309]: done

File: Content_Portal/Documentation/Assets/toc.yml (hunk 8 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/toc.yml:4]: I'll make this change in my next PR.

**Pull request merged**